packages/console/src/pages/siteVersionPreviewPage.tsx

```diff
@@ -11,12 +11,11 @@ import { NavigationBar } from '../components/navigationBar';
 import { AssetFile, PresignedUpload, Site, SiteVersion, SiteVersionEntry } from '../everypageClient';
 import { useGlobals } from '../globalsContext';
 
-export interface ISiteVersionPreviewPageProps {
-  slug: string;
-  siteVersionId: string;
-}
+export const SiteVersionPreviewPage = (): React.ReactElement => {
+  const routeParams = useRouteParams();
+  const slug = routeParams.accountId as string;
```

Comment:
you are not using the right param here - have you tested this stuff locally cos id be shocked if anything here works?